ped/CPed.hpp

```diff
@@ -16,18 +17,27 @@
 class CPed : public rage::CPhysical
 {
 public:
-	char gap30C[20];
-	rage::fvector3 m_velocity; //0x0320
-	char pad_032C[2564]; //0x032C
-	class CVehicle *m_vehicle; //0x0D30
-	char pad_0D38[912]; //0x0D38
-	class CPlayerInfo *m_player_info; //0x10C8
-	class CPedInventory* m_inventory; //0x10D0
-	class CPedWeaponManager *m_weapon_manager; //0x10D8
-	char pad_10E0[907]; //0x10E0
-	uint8_t m_ped_task_flag; //0x146B
-	char pad_146C[196]; //0x146C
-	float m_armor; //0x1530
+	CPedModelinfo* m_model_info; //0x0020 
```

Comment:
If you inherit from rage::CPhysical, here CPedModelinfo* m_model_info; //0x0020 is not the location of 0x20, but 0x30C